From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: xNombre <kartapolska@gmail.com>
Date: Fri, 23 Jul 2021 11:37:54 +0200
Subject: [PATCH 16/18] init.rc: Don't enable f2fs iostat by default

It is only intended for performance analysis and can
impact I/O throughput.

[MikolajQ: Ported to Android 17: AOSP moved iostat to an ro.debuggable=1 block, which userdebug builds still hit]

Signed-off-by: xNombre <kartapolska@gmail.com>
---
 rootdir/init.rc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 9710f8f..9927ec5 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1294,7 +1294,6 @@ on boot
 
 on boot && property:ro.debuggable=1
     write /dev/sys/fs/by-name/userdata/iostat_period_ms 1000
-    write /dev/sys/fs/by-name/userdata/iostat_enable 1
 
 on nonencrypted
     class_start main
